metricbeat/scripts/generate_imports.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+# Generates the file list.go with all modules and metricsets
+
+header = """/*
+Package include imports all Module and MetricSet packages so that they register
+their factories with the global registry. This package can be imported in the
+main package to automatically register all of the standard supported Metricbeat
+modules.
+*/
+package include
+
+import (
+	// Every module and metricset must be added here so that they can register
```

Comment:
Consider changing this comment in some way to mention that this file is generated and that it should not be hand modified.